recipe/build_cpp.sh

```diff
@@ -1,22 +1,32 @@
-mkdir -p build-cpp; cd build-cpp
-rm -rf CMakeCache.txt
+#!/bin/bash
+
+# Isolate the build.
+mkdir -p build-cpp
+cd build-cpp || exit 1
 
 if [[ "$PKG_NAME" == *static ]]; then
     BUILD_TYPE="-DBUILD_SHARED_LIBS=OFF"
 else
     BUILD_TYPE="-DBUILD_SHARED_LIBS=ON"
 fi
 
-cmake ${CMAKE_ARGS} .. \
-      -DCMAKE_BUILD_TYPE=Release \
```

Comment:
I think we need to add this back @ScottBailey @tobijk

Reply:
I opened https://github.com/AnacondaRecipes/reproc-feedstock/pull/2 to see if this fixes it.